api-spec/openapi.yaml

```diff
@@ -521,7 +521,7 @@ components:
         statusDetails:
           type: string
         eventStatus:
-          $ref: "https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-transactions-service/main/api-spec/v1/transactions-api.yaml#/components/schemas/TransactionStatus"
+          $ref: "https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-transactions-service/13081d3cd4d2a77c4cae82aa67840d17ea4a7dd1/api-spec/v1/transactions-api.yaml#/components/schemas/TransactionStatus"
```

Comment:
TODO to be restored

Reply:
fixed with 75d91b16df473d0132ee7b85100c70919b286190